src/index.js

```diff
@@ -1 +1,30 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import getData from './modules/get.js';
+import postData from './modules/post.js';
+import validate from './modules/validate.js';
+
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/24c0c3c116974ac49488d4eb0267bbc3/scores/';
+
+const refreshBtn = document.querySelector('#refreshBtn');
+refreshBtn.addEventListener('click', () => {
+  getData(url);
+});
+
+const form = document.querySelector('form');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const nameInput = document.querySelector('#name').value;
+  const scoreInput = document.querySelector('#score').value;
+
+  if (!nameInput || !scoreInput) {
+    validate();
+  }
+
+  const data = {
+    user: nameInput.trim(),
+    score: scoreInput,
+  };
+
```

Comment:
![license-link](https://user-images.githubusercontent.com/93039359/186430889-0636371e-498e-44bf-9412-e9157f5a9542.png)

- [ ] You have added a great readme but you didn't add a working link to your LICENSE file. Please make your link as given below :arrow_down:

`This project is licensed by [MIT](./LICENSE).`

Reply:
Hello @HAMMAS-SALEEM!

Is this not supposed to have been an optional suggestion?
I really do not understand how this is a major blocker on why this PR shouldn't be merged pls I need further clarifications.

Reply:
@DuaneDave You should have a descriptive readme file. So this should not be in optional.
Please see the the requirement which says **Document your work in a professional way**

Reply:
Okay